trade_management_unit/models/UserConfiguration.py

```diff
@@ -1,21 +1,203 @@
+    )
+    
+    # Stop loss and take profit settings
+    default_stop_loss_percentage = models.FloatField(
+        default=2.0, 
+        validators=[MinValueValidator(0.1), MaxValueValidator(50.0)],
+        help_text='Default stop loss percentage'
+    )
+    default_take_profit_percentage = models.FloatField(
+        default=4.0, 
+        validators=[MinValueValidator(0.1), MaxValueValidator(100.0)],
+        help_text='Default take profit percentage'
+    )
+    
+    # Status and metadata
+    is_active = models.BooleanField(default=True, db_index=True, help_text='Whether configuration is active')
+    created_at = models.DateTimeField(default=timezone.now)
+    updated_at = models.DateTimeField(auto_now=True)
+
     class Meta:
         db_table = "user_configurations"
+        verbose_name = "User Configuration"
+        verbose_name_plural = "User Configurations"
+        ordering = ['user_id']
+        constraints = [
+            models.CheckConstraint(
+                check=models.Q(min_reward_risk_ratio__lte=models.F('max_reward_risk_ratio')),
+                name='chk_reward_risk_ratio_order'
+            ),
+            models.CheckConstraint(
+                check=models.Q(default_stop_loss_percentage__lte=models.F('default_take_profit_percentage')),
+                name='chk_stop_loss_take_profit_order'
+            ),
+        ]
 
+    def __str__(self):
+        return f"Configuration for User {self.user_id}"
 
     @classmethod
     def get_attribute(cls, user_id, attribute):
+        """Legacy method for backward compatibility"""
         try:
-            user_config = cls.objects.get(user_id=user_id)
+            user_config = cls.objects.get(user_id=user_id, is_active=True)
             return getattr(user_config, attribute, None)
         except cls.DoesNotExist:
             return None
+
+    @classmethod
+    def get_or_create_for_user(cls, user_id, **kwargs):
+        """Get or create configuration for a user with default values"""
+        defaults = {
+            'risk_appetite': 5.0,
+            'min_reward_risk_ratio': 2.0,
+            'max_reward_risk_ratio': 20.0,
+            'trades_per_session': 100,
+            'max_daily_trades': 500,
+            'max_position_size': Decimal('100000.00'),
+            'position_size_percentage': 10.0,
+            'default_stop_loss_percentage': 2.0,
+            'default_take_profit_percentage': 4.0,
+            'is_active': True,
+        }
+        defaults.update(kwargs)
+        
+        return cls.objects.get_or_create(
+            user_id=user_id,
+            defaults=defaults
+        )
+
+    @classmethod
+    def get_active_config(cls, user_id):
+        """Get active configuration for a user"""
+        try:
+            return cls.objects.get(user_id=user_id, is_active=True)
+        except cls.DoesNotExist:
+            return None
+
+    def calculate_position_size(self, account_balance, instrument_price):
```

Comment:
def calculate_position_size(self, account_balance, instrument_price): 
 trade_management_unit/models/UserConfiguration.py
 
 This method should not be part of the model rather they should be defined in the risk manager utility which we will find later for now let's remove it from here and all the related code